jobs-builder/publish_jobs.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Use this script to publish the jobs on Jenkins.
+#
+set -e
+
+script_dir="$(realpath $(dirname $0))"
+jobs_dir="$script_dir/jobs"
+readonly cmd="jenkins-jobs"
+test_only=0
+config_file=""
+
+function die
+{
+        local msg="$*"
+        echo "ERROR: $msg" >&2
+        exit 1
+}
+
+function usage
+{
+        cat <<-EOF
```

Comment:
These lines don't start with a tab but the rest of the indented lines do (which explains the odd indentation for `cat` on line 26).